source/simulator/checkpoint_restart.cc

```diff
@@ -39,11 +40,25 @@ namespace aspect
     void move_file (const std::string &old_name,
                     const std::string &new_name)
     {
-      const int error = system (("mv " + old_name + " " + new_name).c_str());
+      int error = system (("mv " + old_name + " " + new_name).c_str());
 
-      AssertThrow (error == 0, ExcMessage(std::string ("Can't move files: ")
-                                          +
-                                          old_name + " -> " + new_name));
+      // If the above call failed, e.g. because there is no command-line
+      // available, try with internal functions.
+      if (error != 0)
+        {
+          if (Utilities::fexists(new_name))
+            {
+              error = remove(new_name.c_str());
+              AssertThrow (error == 0, ExcMessage(std::string ("Unable to remove file: "
+                                                               + new_name
+                                                               + ", although it seems to exist.")));
+            }
+
+          error = rename(old_name.c_str(),new_name.c_str());
```

Comment:
I think we decided against using it for temporary visualization files for exactly the reason you mentioned (I think I wanted to use it and you pointed out the problem :smile:, so you saved yourself some trouble here). Currently all places should first try the `system()` command, and then have a fallback if that does not work.